### DISCUSSION TITLE
PIP Availability and Version

### DISCUSSION THREAD

**Discussion by EGI-ILM** [tdviet/fedcloudclient#133] [Q&A]
When is the latest version (with the recently implemented "vo" subcommand) going to be available through PIP?
Note: I did not change the version number anywhere when I did my PR, I am not sure how this is being managed in the project.

**Comment by tdviet** [comment]
PIP will be created and published automatically for every release (from the main branch).  Usually I create a new release if new features are added or significant bugs are fixed.

I will make a release now.

**Reply by EGI-ILM** [reply]
Great, thanks. So it is a manual trigger from Github. And is the version number fixed automatically or do you set it manually?

**Comment by tdviet** [comment]
New version (fedcloudclient 1.2.15) released and published https://pypi.org/project/fedcloudclient/.

It would help if you or Enol can add documentation for the new command "fedcloud endpoint vos" in this file: https://github.com/tdviet/fedcloudclient/blob/master/docs/usage.rst

**Reply by EGI-ILM** [reply]
OK, I will talk to him
